### ISSUE TITLE
Firestore collections get realtime update

### ISSUE THREAD

**Issue by uzarsalan** [EddyVerbruggen/nativescript-plugin-firebase#558]
Hi! Thanks for this awesome plugin! It's possible to add **firestore().collection(path: string).onSnapshot(doc => void)** function? I wonder that you didn't listed it in Firestore tasks.

**Comment by EddyVerbruggen** [comment]
Main reason is it doesn't exist: https://firebase.google.com/docs/firestore/query-data/listen

**Comment by uzarsalan** [comment]
https://firebase.google.com/docs/firestore/query-data/listen#listen_to_multiple_documents_in_a_collection

**Comment by EddyVerbruggen** [comment]
Yes I know, but that's not `firestore().collection(path: string).onSnapshot(doc => void)`. Listening to multiple docs in that way is not yet implemented in this plugin.

**Comment by ITServicesAM** [comment]
Hi @EddyVerbruggen any progress on this feature? 

BTW: the chaining of collection and document refs works like a charm now :)

**Comment by EddyVerbruggen** [comment]
Nope, haven't got time to look at it yet. Rest assured that if there is an update I will post it here.

**Comment by AndreasHald** [comment]
In need of this feature aswell

**Comment by otnielgomez** [comment]
Also need this feature, this plugin is gold!
